include/mbedtls/ssl.h

```diff
@@ -168,6 +168,8 @@
 #define MBEDTLS_ERR_SSL_VERSION_MISMATCH                  -0x5F00
 /** Invalid value in SSL config */
 #define MBEDTLS_ERR_SSL_BAD_CONFIG                        -0x5E80
+/**< Invalid array size */
+#define MBEDTLS_ERR_SSL_INVALID_ARRAY_SIZE                -0x5E00
```

Comment:
Does this merit a separate error code? Once in the API, we can't take them out. Can we use `MBEDTLS_ERR_SSL_BAD_CONFIG`?

Reply:
Amended